Trakt/Trakt.csproj

```diff
@@ -1,25 +0,0 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I know this PR was abandoned, but for posterity, this change would've sucked immensely for developers as without a .csproj file, the .sln cannot be opened in VS/Rider. They would've been required to 'build' a .csproj with Python first.